### ISSUE TITLE
Latest snap package cannot build a charm

### ISSUE THREAD

**Issue by jnsgruk** [canonical/charmcraft#322]
The latest release to the `beta` channel cannot build a charm out of the box:

[charmcraft-log.txt](https://github.com/canonical/charmcraft/files/6390446/charmcraft-log.txt)

Reproduce with:

```bash
multipass launch --name charmcraft-test
multipass exec charmcraft-test -- bash -c "sudo snap install charmcraft --beta && \
mkdir mycharm && \
cd mycharm && \
charmcraft init && \
charmcraft build"
```

Error from the `charmcraft build` stage:

```
Executing ['pip3', 'list'] failed with return code 1
problems using pip (full execution logs in /home/ubuntu/snap/charmcraft/common/charmcraft-log-pg68k6ko)
```

Log file is attached to this issue.

**Comment by ianmjones** [comment]
I'm having the same problem with charmcraft snap beta 0.10.1.

I had to `snap revert charmcraft` to switch back to 0.10.0 locally, but my CI builds are failing as they use latest snap version.

See end of the following build script: https://builds.sr.ht/~ianmjones/job/497025#task-build

Locally I'm using elementaryOS (Ubuntu 18.04 base), but in CI it's Ubuntu 20.04.

**Comment by jnsgruk** [comment]
Fixed with `0.10.2` release.
